pyabc/visualization/confidence.py

```diff
@@ -118,7 +118,7 @@ def plot_confidence_intervals(
                 kde_max_1d[i_par, i_t] = _kde_max_1d_pnt[par]
             # confidences
             for i_c, confidence in enumerate(confidences):
```

Comment:
Maybe we can rename the `confidences` argument to something better? Maybe `levels`?

Reply:
Done.